rust/src/address.rs

```diff
@@ -121,25 +122,42 @@ impl Address {
                 buf.push(header);
                 buf.extend(enterprise.payment.to_bytes());
             },
+            AddrType::Reward(reward) => {
+                let header: u8 = 0b1110_0000
+                                | (reward.payment.kind() << 4)
+                                | (reward.network & 0xF);
+                buf.push(header);
+                buf.extend(reward.payment.to_bytes());
+            },
         }
         println!("to_bytes({:?}) = {:?}", self, buf);
         buf
     }
 
-    pub fn from_bytes(data: Vec<u8>) -> Result<Self, JsValue> {
+    pub fn from_bytes(data: Vec<u8>) -> Result<Address, JsValue> {
```

Comment:
This was independent but needed when calling from wasm. wasm_bindgen can't figure out the `Self` when exporting to wasm even though here `Self` resolves to `Address`.